sequencing/sequencer.go

```
@@ -32,235 +31,13 @@
-}
-
-func totalBytes(data [][]byte) int {
-	total := 0
-	for _, sub := range data {
-		total += len(sub)
-	}
-	return total
-}
+// Key prefixes for BadgerDB
+var (
+	keyLastBatchHash   = []byte("lastBatchHash")
```

golangci-lint (1.62.2)

var `keyLastBatchHash` is unused

(unused)